src/SDKErrors.js

```diff
@@ -104,3 +104,4 @@ E('ERROR_DELETE_IP_ALLOWLIST_BINDING', 'Could not remove IP Allow List binding:
 E('ERROR_UNSUPPORTED_ADVANCE_STEP', 'Advancing the step %s is not supported in the CLI at present.')
 E('ERROR_REFRESH_STEP_STATE', 'Cannot refresh step state: %s')
 E('ERROR_STEP_STATE_NOT_RUNNING', 'The %s step in execution %s is not currently running.')
+E('ERROR_POST_COMMERCE', 'Could not post to Commerce endpoint %s')
```

Comment:
Since this error is specific to CLI commands, please use an enum name and value that reference CLI commands.

Reply:
Made requested changes to the enum name and value